### PR TITLE
Adding Errata incremental update docs

### PR DESCRIPTION
**Pull request by waldenraines**

### CODE REVIEW

**Comment by jlsherrill** [comment]
I would write more about what an incremental update actually is, and what the implications are.

**Comment by waldenraines** [comment]
@jlsherrill updated, see what you think.

File: docs/user_guide/errata/index.md [hunk 1]
```diff
@@ -70,3 +78,44 @@ Note that you can filter by Lifecycle Environment and Content View.
 
 ![View Repositories Containing Errata](./errata_repositories.png)
 
+## Applying Errata
+
+How Errata is applied to a Content Host(s) depends on whether the Errata is installable.
+
+- If the Errata is already installable then the Errata is applied to the Content Host(s).
+- If the Errata is not installable then an Incremental Update is generated.  An Incremental Update creates a point release of the Content View with the
+Errata included.  The Errata can also be applied to the Content Host(s) as part of this process.
```
Comment (jlsherrill) [inline docs/user_guide/errata/index.md]: extra newline

**Comment by waldenraines** [comment]
@jlsherrill updated.

File: docs/user_guide/errata/index.md [hunk 2]
```diff
@@ -6,17 +6,24 @@ sidebar: sidebars/documentation.html
...
-Errata are incremental updates between major releases.  An Erratum is metadata about a group of packages that explains the importance of the package updates.
+Errata are updates between major releases.  An Erratum is metadata about a group of packages that explains the importance of the package updates.
 Errata may be released individually on an as-needed basis or aggregated as a minor release.  There are three main types of errata:
 
 - Enhancement:  the new packages contain one or more added features
 - Bugfix: the new packages contain one or more bug fixes
 - Security:  the new packages fix one or more security vulnerabilities
 
+With regard to Content Hosts, Errata is divided into two distinct classifications depending on whether or not the Errata is present in the
+Content Host's Lifecycle Environment and Content View:
```
Comment (jlsherrill) [inline docs/user_guide/errata/index.md]: this looks like it wrapped too?

**Comment by waldenraines** [comment]
@jlsherrill updated.

File: docs/user_guide/errata/index.md [hunk 3]
```diff
@@ -6,17 +6,23 @@ sidebar: sidebars/documentation.html
...
-Errata are incremental updates between major releases.  An Erratum is metadata about a group of packages that explains the importance of the package updates.
-Errata may be released individually on an as-needed basis or aggregated as a minor release.  There are three main types of errata:
+Errata are updates between major releases.  An Erratum is metadata about a group of packages that explains the importance of the package updates.  Errata may be released individually on an as-needed basis or aggregated as a minor release.  There are three main types of errata:
 
 - Enhancement:  the new packages contain one or more added features
 - Bugfix: the new packages contain one or more bug fixes
 - Security:  the new packages fix one or more security vulnerabilities
 
+With regard to Content Hosts, Errata is divided into two distinct classifications depending on whether or not the
+Errata is present in the Content Host's Lifecycle Environment and Content View:
```
Comment (jlsherrill) [inline docs/user_guide/errata/index.md]: this is still not wrapped properly

**Comment by waldenraines** [comment]
@jlsherrill for real this time.

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**